migration/idempotent/001-base.sql

```diff
@@ -866,7 +866,7 @@ AS
 $$
 BEGIN
     EXECUTE FORMAT(
-        'UPDATE prom_data_series.%1$I SET delete_epoch = current_epoch+1 FROM _prom_catalog.ids_epoch WHERE delete_epoch IS NULL AND id = ANY($1)',
+        'UPDATE prom_data_series.%1$I s SET delete_epoch = current_epoch FROM _prom_catalog.ids_epoch WHERE s.delete_epoch IS NULL AND id = ANY($1)',
```

Comment:
1. Is it possible to rename the `s.delete_epoch` column? I would prefer to if there are no technical issues with it.
2. I don't think this is correct w.r.t our locking strategy. Same as in the mark function, this need at least a `SELECT ... FOR SHARE` on `ids_epoch`.

Reply:
1. Yeah, that would be the best, in my opinion.
2. 👍🏼

Reply:
@sumerman I've hit an issue: the upgrade tests are not happy, because the index definition is not the same on the "fresh install" and "update" paths. The reason is that the attribute which belongs to the index is not renamed. I could reach into pg_attribute and rename this, but I'm wondering if it's worth it. Can you take a look at what I have already and let me know.